games/devtest/mods/lighting/set_lightIntensity.lua

```diff
@@ -0,0 +1,141 @@
+
+local lightIntensity_sections = {
+  {n = "colorOffset", d = "Color offset",
+    entries = {
+      {n = "r", d = "Red color offset", min = -1, max = 2},
+      {n = "g", d = "Green color offset", min = -1, max = 2},
+      {n = "b", d = "Blue color offset", min = -1, max = 2},
+    }
+  },
+  {n = "colorRatioCoef", d = "Color day-night ratio coefficient",
+    entries = {
+      {n = "r", d = "Red color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+      {n = "g", d = "Green color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+      {n = "b", d = "Blue color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+    }
+  }
+}
+
+local function dump_lightIntensity(lightIntensity)
+	local result = "{\n"
+	local section_count = 0
+	for _,section in ipairs(lightIntensity_sections) do
+		section_count = section_count + 1
+
+		local parameters = section.entries or {}
+		local state = lightIntensity[section.n] or {}
+
+		result = result.."  "..section.n.." = {\n"
+
+		local count = 0
+		for _,v in ipairs(parameters) do
+			count = count + 1
+			result = result.."    "..v.n.." = "..(math.floor(state[v.n] * 1000)/1000)
+			if count < #parameters then
+				result = result..","
+			end
+			result = result.."\n"
+		end
+
+		result = result.."  }"
+
+		if section_count < #lightIntensity_sections then
+			result = result..","
+		end
+		result = result.."\n"
+	end
+	result = result .."}"
+	return result
+end
+
+minetest.register_chatcommand("set_lightIntensity", {
+	params = "",
+	description = "Tune lighting lightIntensity parameters",
+	func = function(player_name, param)
+		local player = minetest.get_player_by_name(player_name);
```

Comment:
Please no semicolons in Lua (unless necessary, which is very rarely)

Reply:
fixed